packages/loader/lib/main.js

```diff
@@ -59,6 +64,8 @@ var mainContext = this;
       for (var i = 0; i < length; i++) {
         if (deps[i] === 'exports') {
           reified[i] = exports;
+        } else if (deps[i] === 'require') {
```

Comment:
sorta, once we upgrade the loader we will get this for free. Also that require should actually resolve relative to the current module

This is merely a stepping stone to align with loader.js https://github.com/ember-cli/loader.js/blob/master/loader.js#L93-L99